packages/common/core-interfaces/src/test/types/fluidObjectTypes.ts

```diff
@@ -7,7 +7,8 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 
-import { IFluidLoadable, IProvideFluidLoadable, FluidObject, FluidObjectKeys } from "../../";
+import type { IProvideFluidLoadable, FluidObject, FluidObjectKeys } from "../../";
+import { IFluidLoadable } from "../../";
 
```

Comment:
Shouldn't this be a type import?

Reply:
nm - see the uses on lines 67,68

Reply:
IFluidLoadable is a const.

```ts
export const IFluidLoadable: keyof IProvideFluidLoadable = "IFluidLoadable";
```

Reply:
Right, I was just trying to explain why it couldn't be imported as a type only - but I guess the real answer is that it's USED as a const in this file, not that it is defined as both a const and a type.